### PR TITLE
feat(contribution-types): adds mentoring

### PR DESCRIPTION
**Pull request by mloning**
**What**:
Add support for mentoring (see https://github.com/all-contributors/all-contributors/issues/423)

**Why**:

**How**:
Added mentoring to [`src/util/contribution-types.js`](https://github.com/all-contributors/all-contributors-cli/blob/master/src/util/contribution-types.js)

**Checklist**:

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged 
- [ ] Added myself to contributors table 

**Comments**:
If tests are needed, could you point me to the files that I have to change/add?

### CODE REVIEW

File: src/util/contribution-types.js [hunk 1]
```diff
@@ -74,6 +74,10 @@ const defaultTypes = function(repoType) {
       symbol: '🚧',
       description: 'Maintenance',
     },
+    mentoring: {
+      symbol: '🧑‍🏫',
```
Comment (Berkmann18) [inline src/util/contribution-types.js:78]: It's bizarre that the emoji shows up as the 2 emojis when on the issue it was different (viewing from the same browser, OS and device).

Comment (mloning) [reply src/util/contribution-types.js:78]: Yes had the same thing on the app yesterday and in PyCharm in the markdown file but not the js file

Comment (Berkmann18) [reply src/util/contribution-types.js:78]: Also, is there a reason behind adding a space after `function` which is not done anywhere in the codebase (as far as arrow functions go)?

Comment (mloning) [reply src/util/contribution-types.js:78]: Yes I also saw that, did do that on purpose, must have been an auto formatting thing, I'll remove them again

Comment (mloning) [reply src/util/contribution-types.js:78]: they seem to get added by the pre-commit tasks? I remove the white spaces, I see them in `git diff` but when I run `git add` and `git commit` it tells me that my commit is empty ... I've changed on GitHub now

Comment (Berkmann18) [reply src/util/contribution-types.js:78]: Oh okay.

**Comment by Berkmann18** [comment]
@all-contributors Please add @mloning for code

**Pull request merged**